src/java/org/apache/cassandra/index/sai/analyzer/JSONAnalyzerParser.java

```diff
@@ -99,4 +104,15 @@ private static Analyzer matchBuiltInAnalzyer(String maybeAnalyzer) {
         }
         return null;
     }
+
+    private static Map<String, String> applyTokenizerDefaults(String filterName, Map<String, String> args)
```

Comment:
TODO: do we need to consider filters?

Reply:
No, filters do not have defaults in lucene, so configuring defaults would not matter.